johnsonandjohnson/Bodiless-JS
# Fixing periodically get 500 error issue

## Changes

Fixing git diff-files check index issue, replace diff-files with `git diff`.

git diff-files is one of low-level (plumbing) commands, which does not update index comparing "git diff". The Lerna bootstrap process updates some file's modification time without content change. In these cases, the "git diff-files" can not reflect the right status of file updates and result in error on follow up commit.

## Test Instructions

- Start app with `npm run setup`
- Go to edit UI and click "Pull"

Expect: no errors.

## Related Issues

fixing: #363
